inlanecruising_plugin/include/inlanecruising_plugin/inlanecruising_config.h

```diff
@@ -29,19 +29,13 @@ struct InLaneCruisingPluginConfig
                                            // Corresponds to saving each nth point.
   double minimum_speed = 2.2352;           // Minimum allowable speed in m/s
   double max_accel = 1.5;                  // Maximum allowable longitudinal acceleration in m/s^2
-  
-  double minimum_lookahead_distance = 5.0; // Minimum value for lookahead distance in m
-  double maximum_lookahead_distance = 25.0;// Maximum value for lookahead distance in m
-  double minimum_lookahead_speed = 2.8;    // Minimum speed value for lookahead calculation in m/s
-  double maximum_lookahead_speed = 13.9;   // Maximum speed value for lookahead calculation in m/s
-  double lookahead_ratio = 2.0;            // ratio to calculate lookahead distance from speed
-  
-
+  int lookahead_count = 8;                 // Number of points to look ahead for speed reduction.
   double lateral_accel_limit = 1.5;        // Maximum allowable lateral acceleration m/s^2
-  int moving_average_window_size = 5;      // Size of the window used in the moving average filter to smooth both the
+  int moving_average_window_size = 3;      // Size of the window used in the moving average filter to smooth both the
```

Comment:
Looks like we might need to change the value in the yaml file to match this or vice versa based on testing.

Reply:
matched it